### ISSUE TITLE
Add message icon to rows that do contain a memo

### ISSUE THREAD

**Issue by skironDotNet** [PIVX-Project/PIVX#2888]
Since there is no room to display a memo, maybe you could add an icon next to address to indicate that the transfer includes memo

Here is my primitive drawing
![image](https://github.com/PIVX-Project/PIVX/assets/2981715/1afe0dd4-8fea-4d66-9ca2-3e314dd83f0a)

**Comment by skironDotNet** [comment]
I don't think that using different color is intuitive type of a UI, because you need to know about it, like to read the instruction "if a color is X then it means you have memo", while using some "now days" UI "industry standard" where the user can quickly tell, "oh there is some message inside" by looking at the icon. 

The demo I used was just quick grab from google pics, so it's not the icon I desire, but just to show that having "message indicator" style icon is easy to discover and understand, the user would likely click/tap on the icon, and normal row popup would occur with all the details and 90% chance he would look at the bottom to notice memo text and from now on the user intuitively discovered the fact that "icon means memo is there + tap on the row to see it" and having approach with different row color doesn't do that, it would be more like "why is this row a different color?"

**Comment by Liquid369** [comment]
The current idea is, with the color it would make you re-read said transaction text which then would indicate by saying “received shielded memo” but yes an icon is still absolutely needed and necessary but the icons I had proposed fell semi flat from “generics”. It will still make enough for a user to understand and differentiate
